docs/runbooks/e2e-required-check.md

```
@@ -0,0 +1,87 @@
+
+Beachte: `strict: true` bedeutet, dass Checks auch bei neueren Commits erneut erforderlich sind.
+
+## Absicherung gegen Flakes
+
+**Erst nach mehreren grünen Läufen erzwingen.** Der Workflow ist neu reaktiviert; gib dem Team Zeit, um zu verifizieren, dass die Checks stabil grün sind.
+
+## Rollback
+
+Trigger wieder deaktivieren oder Required-Check-Häkchen entfernen:
+
+### Option A: Trigger deaktivieren (`.github/workflows/e2e-smokes.yml`)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)